client/my-sites/plans-features-main/components/free-free-dialog.tsx

```diff
@@ -0,0 +1,290 @@
+	const rawPrice = useSelector( ( state ) => getPlanRawPrice( state, planProductId, true ) );
+
+	return (
+		<Dialog
+			isBackdropVisible={ true }
+			isVisible={ true }
+			onClose={ onClose }
+			showCloseIcon={ true }
+		>
+			<Global
+				styles={ css`
+					.dialog__backdrop.is-full-screen {
+						background-color: rgba( 0, 0, 0, 0.6 );
+					}
+					.ReactModal__Content--after-open.dialog.card {
+						border-radius: 4px;
+						width: 605px;
+					}
+				` }
+			/>
+			<DialogContainer>
+				<Heading>{ translate( "Don't miss out" ) }</Heading>
+				<TextBox>
+					{ translate( "With a Free plan, you'll miss out on a lot of great features:" ) }
+				</TextBox>
+				<List>
+					<ListItem>
+						<div>
+							<CrossIcon icon="cross" size={ 24 } />
+						</div>
+						<TextBox>
+							{ translate(
+								'{{strong}}No free custom domain:{{/strong}} Your site will be shown to visitors as {{strong}}%(freeSubdomain)s{{/strong}}',
+								{
+									args: {
+										freeSubdomain,
+									},
+									components: { strong: <strong></strong> },
+								}
+							) }
+						</TextBox>
+					</ListItem>
+					<ListItem>
+						<div>
+							<CrossIcon icon="cross" size={ 24 } />
+						</div>
+						<TextBox>
+							{ translate(
+								'{{strong}}No ad-free experience:{{/strong}} Your visitors will see external ads on your site.',
+								{
+									components: { strong: <strong></strong> },
+								}
+							) }
+						</TextBox>
+					</ListItem>
+					<ListItem>
+						<div>
+							<CrossIcon icon="cross" size={ 24 } />
+						</div>
+						<TextBox>
+							{ translate(
+								'{{strong}}No unlimited professional customer support{{/strong}} (only community forums)',
+								{
+									components: { strong: <strong></strong> },
+								}
+							) }
+						</TextBox>
+					</ListItem>
+					<ListItem>
+						<div>
+							<CrossIcon icon="cross" size={ 24 } />
+						</div>
+						<TextBox>
+							{ translate(
+								'{{strong}}No extra storage:{{/strong}} You only get 1GB for photos, videos, media, and documents.',
+								{
+									components: { strong: <strong></strong> },
+								}
+							) }
+						</TextBox>
+					</ListItem>
+				</List>
+				<TextBox>
+					{ rawPrice &&
+						translate(
+							'Unlock {{strong}}all of{{/strong}} these features with a Personal plan,starting at just %(planPrice)s/month, {{break}}{{/break}} with a 14-day money back guarantee.',
+							{
+								args: {
+									planPrice: formatCurrency( rawPrice, currencyCode, {
+										stripZeros: true,
+									} ),
+								},
+								components: { break: <br />, strong: <strong></strong> },
+							}
+						) }
+				</TextBox>
+				<TextBox>
+					{ translate(
+						'As a bonus, you will get a custom domain - like {{strong}}{{italic}}yourgroovydomain.com{{/italic}}{{/strong}} - {{break}}{{/break}} free for the first year ({{suggestionPrice}}{{/suggestionPrice}} value).',
```

Comment:
Initially this was the plan but later it was decided to go with a static url. I left in the hook because I am betting that we might need to use it in the future for some sort of upsell 🤔